src/deposit/WithdrawalForm.js

```diff
@@ -6,66 +6,73 @@ import InputGroup from '../_common/InputGroup';
 import SelectGroup from '../_common/SelectGroup';
 import Modal from '../containers/Modal';
 import currencies from '../_constants/currencies';
+import * as LiveData from '../_data/LiveData';
 
 export default class WithdrawForm extends Component {
     static propTypes = {
-        selectedPaymentAgent: PropTypes.string,
-        paymentAgents: PropTypes.array.isRequired,
+        paymentAgent: PropTypes.object.isRequired,
         currency: PropTypes.oneOf(currencies).isRequired,
         actions: PropTypes.object,
-        withdrawAmount: PropTypes.number,
-        withdrawFailed: PropTypes.bool,
-        withdrawError: PropTypes.string,
-        withdrawClicked: PropTypes.bool,
-        confirmClicked: PropTypes.bool,
-        dryRunFailed: PropTypes.bool,
-        dryRunError: PropTypes.string,
-        inProgress: PropTypes.bool,
+        email: PropTypes.string,
     };
-
+    async componentWillMount() {
+        const { email } = this.props;
+        await LiveData.api.sendVerificationEmail(email);
```

Comment:
Thought as well. But considering it does not return a response. Where should we put it ?